libraries/AP_Motors/AP_Motors_Class.h

```diff
@@ -342,6 +342,11 @@ class AP_Motors {
     // return string corresponding to frame_type
     virtual const char* get_type_string() const { return ""; }
 
+    // output_test_seq - spin a motor at the pwm value specified
+    //  motor_seq is the motor's sequence number from 1 to the number of motors on the frame
+    //  pwm value is an actual pwm value that will be output, normally in the range of 1000 ~ 2000
+    virtual void _output_test_seq(uint8_t motor_seq, int16_t pwm) = 0;
```

Comment:
Since this is public I think you should give this a better name rather than just adding an _

Reply:
its protected not public, so nothing else should be calling it since there are no friend classes.